python/packages/autogen-ext/src/autogen_ext/agents/video_surfer/_action_space.py

```diff
@@ -0,0 +1,158 @@
+import base64 
+import ffmpeg
+import cv2
+import whisper
+import openai
+
+
+def extract_audio(video_path: str, audio_output_path: str) -> str:
```

Comment:
To be included in a separate PR, but I believe it is essential to create unit tests for these tools. Consider data generation approach to generate tiny videos that can be used for tests.

Reply:
okay!